src/translations/en/commerce.php

```diff
@@ -383,7 +385,7 @@
     'Don’t apply any subsequent discounts to an order if this discount is applied' => 'Don’t apply any subsequent discounts to an order if this discount is applied',
     'Don’t apply this tax when the address has a valid VAT ID' => 'Don’t apply this tax when the address has a valid VAT ID',
     'Download PDF' => 'Download PDF',
-    'Download as Zip file' => 'Download as Zip file',
+    'Download PDF…' => 'Download PDF…',
```

Comment:
Is it important to have `Download PDF` _and_ `Download PDF…` strings?

Reply:
Yeah it is weird, one `Download PDF` is used for a button with a dropdown arrrow next to it, and the other  `Download PDF…` acts like the export HUD on the element index, so I used the same convention there.